packages/firefox.rb

```diff
@@ -39,10 +47,22 @@ class Firefox < Package
   depends_on 'pulseaudio'
   depends_on 'sommelier'
 
+  def self.patch   
+    @_wrapper = <<~EOF
+      #!/bin/sh
+      # To get sound working, used : https://codelab.wordpress.com/2017/12/11/firefox-drops-alsa-apulse-to-the-rescue/
+      
+      exec apulse #{CREW_PREFIX}/lib/firefox/firefox "$@"
```

Comment:
I thought we didn't need apulse to get sound working.  In any event, that should be a dependency and it's not listed.  See if it works without the wrapper script.

Reply:
You're right, the sound is working without `apulse`